libyul/backends/evm/StackHelpers.h

```diff
@@ -65,7 +65,6 @@ inline std::string stackToString(Stack const& _stack, Dialect const& _dialect)
 // in the interface below.
 // Based on that information the shuffler decides which is the next optimal operation to perform on the stack
 // and calls the corresponding entry point in the shuffling operations (swap, pushOrDupTarget or pop).
-/*
 template<typename ShuffleOperations>
 concept ShuffleOperationConcept = requires(ShuffleOperations ops, size_t sourceOffset, size_t targetOffset, size_t depth) {
```

Comment:
Hooray for C++20!

Reply:
We can now remove `Can be defined as actual concept once we switch to C++20.` from the comment above :)